go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/onsi/ginkgo v1.12.0 // indirect
 	github.com/onsi/gomega v1.9.0 // indirect
 	github.com/pkg/errors v0.8.1
-	github.com/privacybydesign/gabi v0.0.0-20210409092845-6113e0d3ec81
+	github.com/privacybydesign/gabi v0.0.0-20210520084001-626ad7db13a6
```

Comment:
We use features from go 1.15 (URL.RawFragment), update go version from 1.13 to 1.15 here?